packages/web/src/components/usdc-manual-transfer-modal/USDCManualTransferModal.module.css

```diff
@@ -0,0 +1,29 @@
+.modal {
+  width: 720px;
+}
+
+.modalHeader {
+  text-align: center;
+}
+
+.modalHeader.mobile {
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  gap: var(--unit-6);
+}
+
+.buttonContainer {
+  display: flex;
+  gap: var(--unit-2);
+}
+
+.iconUSDC {
```

Comment:
probably
.iconUSDC path {
  fill: x
}

Reply:
also, maybe we should theme this color in

Reply:
fixed color bug with dylan! i think there's potentially a bug with the `stems/Icon` component that is overriding the color that the svg comes with, so just used the svg directly. probably don't want to add to theme since it's always gonna be this color, and then we'll get it right in harmony.

Reply:
sounds good!